src/Components/Components/CardSerie/CardSerie.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
This is a dependency too.

// Dependencies
import React from 'react';
import { Card } from "react-bootstrap";

Always use single quote ' ' for strings.

Reply:
Sip, podés usar un linter para que haga todo eso. O cambiar las rules del prettier